packages/cron-job/MessageHandler/ExecuteCronJobMessageHandler.php

```diff
@@ -19,6 +22,12 @@ public function __construct(
     public function handleExecuteCronJobMessage(ExecuteCronJobMessage $message): void
     {
         if (!($execution = $message->getExecution())->isExecutable(new \DateTimeImmutable())) {
```

Comment:
Should be move this to the    $this->cronJobProcessor->process($execution); ?

What do you think ?